forum/startups/tests.py

```diff
@@ -1,3 +1,27 @@
-from django.test import TestCase
+from rest_framework.test import APITestCase
+from rest_framework.reverse import reverse
+from rest_framework import status
 
-# Create your tests here.
+from forum.tests_setup import TestUserSetupMixin
+from startups.models import StartupSize
+
+
+class StartupsTestCase(TestUserSetupMixin):
+    CREATE_OBJ_COUNT: int = 5
+
+    def setUp(self) -> None:
+        self.setup_test_user()
+
+        for i in range(self.CREATE_OBJ_COUNT):
+            tmp_size_obj = StartupSize.objects.create(
+                name="test size",
+                people_count_min=20,
+                people_count_max=30
+            )
+            StartupSize.objects.get(size_id=tmp_size_obj.size_id)
```

Comment:
Remove the unnecessary retrieval of `StartupSize` objects immediately after creation in the loop.